aws-datastore/src/main/java/com/amplifyframework/datastore/DataStoreConfiguration.java

```diff
@@ -175,6 +176,29 @@ public Integer getSyncPageSize() {
         return this.syncPageSize;
     }
 
+    /**
+     * Returns a QueryPredicate that should be used to filter the data that is synced.
+     * @param modelClass Class of the {@link Model}.
+     * @return a QueryPredicate that should be used to filter the data that is synced.
+     */
+    @NonNull
+    public QueryPredicate getSyncQueryPredicate(Class<? extends Model> modelClass) {
+        DataStoreSyncExpression expression = syncExpressions.get(modelClass);
+        if (expression != null) {
+            return expression.resolvePredicate();
+        }
+        return QueryPredicates.all();
+    }
+
+    /**
+     * Returns the Map of all {@link DataStoreSyncExpression}s used for filtering the data that is synced.
+     * @return the Map of all {@link DataStoreSyncExpression}s used for filtering the data that is synced.
+     */
+    @NonNull
+    public Map<Class<? extends Model>, DataStoreSyncExpression> getSyncExpressions() {
```

Comment:
If you have the one above, do you need this `Map`-returning one, as well?

Reply:
I considered the same question.  The main reason I need this though is for obtaining the `Map` from a user-provided `DataStoreConfiguration`, while merging with the default configuration.

Reply:
Ah yes, that makes sense. 👍